cmd/terraform_native_commands.go

```diff
@@ -0,0 +1,201 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// getTerraformCommands returns an array of statically defined Terraform commands with flags
+func getTerraformCommands() []*cobra.Command {
+	// List of Terraform commands
+	return []*cobra.Command{
+		{
+			Use:   "plan",
+			Short: "Show changes required by the current configuration",
+			Long:  "Generate an execution plan, which shows what actions Terraform will take to reach the desired state of the configuration.",
+		},
+		{
+			Use:   "apply",
+			Short: "Apply changes to infrastructure",
+			Long:  "Apply the changes required to reach the desired state of the configuration. This will prompt for confirmation before making changes.",
+		},
+		{
+			Use:   "workspace",
+			Short: "Manage Terraform workspaces",
+			Long:  "Create, list, select, or delete Terraform workspaces, which allow for separate states within the same configuration.",
+		},
+		{
+			Use:   "clean",
+			Short: "Clean up resources",
+			Long:  "Remove unused or outdated resources to keep the infrastructure clean and reduce costs.",
+		},
+		{
+			Use:   "version",
+			Short: "Show the current Terraform version",
+			Long:  "Displays the current version of Terraform installed on the system.",
+		},
+		{
+			Use:   "varfile",
+			Short: "Load variables from a file",
+			Long:  "Load variable definitions from a specified file and use them in the configuration.",
+		},
+		{
+			Use:   "write varfile",
+			Short: "Write variables to a file",
+			Long:  "Write the variables used in the configuration to a specified file for later use or modification.",
+		},
+		{
+			Use:   "destroy",
+			Short: "Destroy previously-created infrastructure",
+			Long:  "Destroy all the infrastructure managed by Terraform, removing resources as defined in the state file.",
+		},
+		{
+			Use:   "import",
+			Short: "Associate existing infrastructure with a Terraform resource",
+			Long:  "Import existing infrastructure into Terraform management by specifying the resource type and ID.",
+		},
+		{
+			Use:   "refresh",
+			Short: "Update the state to match remote systems",
+			Long:  "Refresh the Terraform state, reconciling the local state with the actual infrastructure state.",
+		},
+		{
+			Use:   "init",
+			Short: "Prepare your working directory for other commands",
+			Long:  "Initialize the working directory containing Terraform configuration files. It will download necessary provider plugins and set up the backend.",
```

Comment:
```suggestion
			Long:  "Initialize the working directory containing Terraform configuration files. It will download necessary provider plugins and set up the backend. Note, that Atmos will automatically call init for you, when running `plan` and `apply` commands.",
```

Reply:
done